beta/src/pages/learn/writing-markup-with-jsx.md

```diff
@@ -170,7 +170,7 @@ This is how Hedy Lamarr's image and list items look closed:
 </>
 ```
 
-### 3. camelCase <s>all</s> most of the things! {#3-camelcase-salls-most-of-the-things}
+### 3. camelCase <s>all</s> most of the things! {/*3-camelcase-salls-most-of-the-things*/}
```

Comment:
This is the only case where we have a tag in heading. shall we skip the tags? or leave it as such?

Reply:
We’d need an actual HTML parser to parse HTML properly, and match exactly how GH does HTML, which gets complex fast (but I could make it work).
If possible, I’d recommend not putting the `<s>` in there.
Alternatively, MDX 2 would remove it from the slugs as well.

Reply:
https://beta.reactjs.org/learn/writing-markup-with-jsx#3-camelcase-salls-most-of-the-things

This heading might actually need a <s> I guess. 
Would it be good if we could remove the opening and closing tags using regex? We might not get complex HTML in heading and this might work?

Reply:
Trying to parse HTML with regexes is... not perfect. The alternative is using `remark-rehype`, `rehype-raw`, and then `rehype-slug`, which works! Just a bunch of code.

However, this exact ID was already there before though, right? I understand what you want, it’ll get complex but maybe it’s good. But I’d think that that could be another PR?

Reply:
Yep Sounds good 😄 !